src/controllers/account/index.ts

```diff
@@ -1,45 +1,68 @@
 import { BaseContext } from 'koa'
-// import AccountUtilities from './account'
+import * as Strings from '../../helpers/strings'
+import * as Methods from '../../helpers/methods'
+import {
+  getStoredVerifierHash,
+  generateAndStoreToken
+} from './account'
 
 export default class AccountController {
-  public static async requestReset (ctx: BaseContext) {
+  // Endpoint to create hash to send in email to the user on login
+  public static async login (ctx: BaseContext) {
```

Comment:
Missing first step in this function: 

Lookup in MySQL that this email address is in fact an active registered user (so we aren't sending login links to non-registered users). Send back 200 to the front end. The front end should say something like "If that email was registered, you should receive a login link within a few seconds." That way we won't leak valid emails.

Reply:
why would we not just sign them up if they weren't already registered

Reply:
so make registration and login the same thing?

Reply:
ah i guess you're right because we need the account first and last name so a diff flow makes sense. done.